services/download/src/main/java/app/coronawarn/server/services/download/config/DownloadServiceConfig.java

```diff
@@ -24,8 +23,9 @@
   @Min(0)
   @Max(28)
   private Integer retentionDays;
-  private TekFieldDerivations tekFieldDerivations;
   private Validation validation;
+  @Autowired
+  private TekFieldDerivations tekFieldDerivations;
```

Comment:
Shall we stay consistent to the other classes and stay with constructor injection instead of field injection?

Reply:
Or I'd suggest to use setter injection since there is already an existing setter.